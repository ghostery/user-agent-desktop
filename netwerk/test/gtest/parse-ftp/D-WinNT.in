ftp> open ftp.microsoft.com
220 Microsoft FTP Service
Name (ftp.microsoft.com:cyp): 
331 Anonymous access allowed, send identity (e-mail name) as password.
230-This is FTP.Microsoft.Com
230 Anonymous user logged in.
Remote system type is Windows_NT.
ftp> quote dirstyle
200 MSDOS-like directory output is on
ftp> ls
200 PORT command successful.
150 Opening ASCII mode data connection for /bin/ls.
02-05-01  05:52PM       <DIR>          20Year
09-26-00  03:20PM       <DIR>          AccessFoxPro
12-21-00  02:39PM       <DIR>          AlbGrp
05-31-01  11:16AM       <DIR>          Alexandra
01-19-01  09:36AM       <DIR>          anna
04-06-00  12:53PM       <DIR>          anz
05-10-00  01:37PM       <DIR>          Chase Bobko2
03-29-00  02:43PM       <DIR>          cnn
11-21-00  11:21AM       <DIR>          Darin
03-09-00  05:55PM       <DIR>          digitalchicago
09-06-00  11:38AM       <DIR>          Dublin
01-30-01  07:15PM       <DIR>          eleanorf
04-26-01  11:04AM       <DIR>          girvin
04-26-00  09:28PM       <DIR>          Hires
08-15-00  05:03PM       <DIR>          HR
10-24-99  01:29AM              4368384 IMG00022.PCD
05-18-01  07:52AM       <DIR>          jacubowsky
10-12-00  10:55AM       <DIR>          JFalvey
03-28-01  02:38PM       <DIR>          johnci
07-14-00  08:59AM       <DIR>          Karin
09-07-00  05:48PM       <DIR>          Kjung
09-28-00  08:59AM       <DIR>          LarryE
08-17-00  06:06PM       <DIR>          Larson
09-12-00  05:22PM       <DIR>          marion
08-09-00  12:30PM       <DIR>          ms25
11-16-00  03:58PM       <DIR>          MS25Brochure
03-29-00  12:07PM       <DIR>          MShistory
09-05-00  10:35AM       <DIR>          Neils
08-02-00  06:25PM       <DIR>          NLM
09-06-00  03:04PM       <DIR>          PageOne
06-27-00  06:00PM       <DIR>          pccomputing
05-09-01  01:27PM       <DIR>          pictures
07-21-00  04:23PM       <DIR>          pranks
08-22-00  10:36AM       <DIR>          Sean
08-10-00  01:54PM       <DIR>          SLeong
09-07-00  05:46PM       <DIR>          svr
10-23-00  01:27PM       <JUNCTION>     b_junction_sample => foo
06-15-00  07:37AM       <JUNCTION>     c_junction_sample -> bar too
07-14-00  01:35PM              2094926 canprankdesk.tif
07-21-00  01:19PM                95077 Jon Kauffman Enjoys the Good Life.jpg
07-21-00  01:19PM                52275 Name Plate.jpg
07-14-00  01:38PM              2250540 Valentineoffprank-HiRes.jpg
01-11-14  12:25AM             18864566 Bug961346
10-10-2014  10:10AM       <DIR>        Bug1061898
01-18-12  19:00             30724571 Bug1125816
226 Transfer complete.
ftp> close
221  Thank-You For Using Microsoft Products!
ftp> 

