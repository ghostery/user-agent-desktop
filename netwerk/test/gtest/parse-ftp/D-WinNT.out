<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

02-05-2001  17:52:00       <DIR>          20Year
09-26-2000  15:20:00       <DIR>          AccessFoxPro
12-21-2000  14:39:00       <DIR>          AlbGrp
05-31-2001  11:16:00       <DIR>          Alexandra
01-19-2001  09:36:00       <DIR>          anna
04-06-2000  12:53:00       <DIR>          anz
05-10-2000  13:37:00       <DIR>          Chase Bobko2
03-29-2000  14:43:00       <DIR>          cnn
11-21-2000  11:21:00       <DIR>          Darin
03-09-2000  17:55:00       <DIR>          digitalchicago
09-06-2000  11:38:00       <DIR>          Dublin
01-30-2001  19:15:00       <DIR>          eleanorf
04-26-2001  11:04:00       <DIR>          girvin
04-26-2000  21:28:00       <DIR>          Hires
08-15-2000  17:03:00       <DIR>          HR
10-24-1999  01:29:00              4368384 IMG00022.PCD
05-18-2001  07:52:00       <DIR>          jacubowsky
10-12-2000  10:55:00       <DIR>          JFalvey
03-28-2001  14:38:00       <DIR>          johnci
07-14-2000  08:59:00       <DIR>          Karin
09-07-2000  17:48:00       <DIR>          Kjung
09-28-2000  08:59:00       <DIR>          LarryE
08-17-2000  18:06:00       <DIR>          Larson
09-12-2000  17:22:00       <DIR>          marion
08-09-2000  12:30:00       <DIR>          ms25
11-16-2000  15:58:00       <DIR>          MS25Brochure
03-29-2000  12:07:00       <DIR>          MShistory
09-05-2000  10:35:00       <DIR>          Neils
08-02-2000  18:25:00       <DIR>          NLM
09-06-2000  15:04:00       <DIR>          PageOne
06-27-2000  18:00:00       <DIR>          pccomputing
05-09-2001  13:27:00       <DIR>          pictures
07-21-2000  16:23:00       <DIR>          pranks
08-22-2000  10:36:00       <DIR>          Sean
08-10-2000  13:54:00       <DIR>          SLeong
09-07-2000  17:46:00       <DIR>          svr
10-23-2000  13:27:00       <JUNCTION>     b_junction_sample -> foo
06-15-2000  07:37:00       <JUNCTION>     c_junction_sample -> bar too
07-14-2000  13:35:00              2094926 canprankdesk.tif
07-21-2000  13:19:00                95077 Jon Kauffman Enjoys the Good Life.jpg
07-21-2000  13:19:00                52275 Name Plate.jpg
07-14-2000  13:38:00              2250540 Valentineoffprank-HiRes.jpg
01-11-2014  00:25:00             18864566 Bug961346
10-10-2014  10:10:00       <DIR>          Bug1061898
01-18-2012  19:00:00             30724571 Bug1125816
